src/routes/game/components/GameOverDialog.vue

```diff
@@ -256,15 +245,10 @@ export default {
       try {
         await this.gameStore.requestRematch({ gameId:this.gameStore.id, rematch: true});
       } catch (e) {
-        this.showSnackbar = true;
-        this.snackBarMessage = 'Error requesting rematch';
+        this.$emit('handle-error', 'game.snackbar.rematch.errorRequesting');
```

Comment:
This feels weird because usually the backend is the one sending back the error, but I noticed the `Rematch` endpoint doesn't actually have any errors to throw. 

@itsalaidbacklife is this something your PR addresses?